pkg/virt-launcher/virtwrap/network/dhcp/dhcp_test.go

```diff
@@ -244,5 +244,18 @@ var _ = Describe("DHCP", func() {
 			}))
 			Expect(options[240]).To(Equal([]byte("private.options.kubevirt.io")))
 		})
+
+		It("Should not set RouterIP or Classless Static Routes options with empty values", func() {
+			ip := net.IP{}
+			dhcpOptions := &v1.DHCPOptions{}
+
+			options, err := prepareDHCPOptions(ip.DefaultMask(), ip, nil, nil, []string{}, 1500, "myhost", dhcpOptions)
```

Comment:
nit: would you use variables to make it clear which attribute is which ?

Reply:
Done